building/configlet/lint.md

```diff
@@ -159,21 +159,15 @@ The `config.json` file should have the following checks:
 - The file must be valid JSON
 - The JSON root must be an object
 - The `"authors"` key is required
-- The `"authors"` value must be an non-empty array
-- The `"authors[].github_username"` key is required
-- The `"authors[].github_username"` key must be a non-empty, non-blank string
-- The `"authors[].github_username"` value is treated case-insensitively
-- The `"authors[].exercism_username"` key is optional
-- The `"authors[].exercism_username"` key must be a non-empty, non-blank string
-- The `"authors[].github_username"` value is treated case-insensitively
+- The `"authors"` value must be a non-empty array
+- The `"authors"` values must be non-empty, non-blank strings
+- The `"authors"` values must not have duplicates
+- The `"authors"` values are treated case-insensitively
 - The `"contributors"` key is optional
-- The `"contributors"` value must be an array
-- The `"contributors[].github_username"` key is required
-- The `"contributors[].github_username"` key must be a non-empty, non-blank string
-- The `"contributors[].github_username"` value is treated case-insensitively
-- The `"contributors[].exercism_username"` key is optional
-- The `"contributors[].exercism_username"` key must be a non-empty, non-blank string
-- The `"contributors[].exercism_username"` value is treated case-insensitively
+- The `"contributors"` value must be a non-empty array
```

Comment:
Requiring that `contributors` is non-empty is a new requirement in this PR, and it's not obvious that this was intended.

Reply:
And it should not have been. This is wrong.